### DISCUSSION TITLE
CPU Freq average summary

### DISCUSSION THREAD

**Discussion by BlueWings76** [LibreHardwareMonitor/LibreHardwareMonitor#757] [Ideas]
Hello LHM Fans,

I have an AMD FX-6350 on ASUS M5A99FX PRO 2.0 board.
Planning to implement FX-9590 when I get my hands on it asap.
I wonder if it's possible to put a summarized CPU Frequency display in the Gadget, where all individual cores freq would be avereged. That would be very useful for me. 
I have version 0.8.7 installed, does it make sense to upgrade to the newest 0.9.0 ?

Thanks, Peter

**Comment by nincode** [comment]
Would be great to have a running average column that could be added next to Min, Max. 
This would entail also adding a setting for the running average window size.

This makes it much easier to see trends over time in spiky data.
